programming_foundations/lean_fridge_app_final_proj/lean_fridge_setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from sqlalchemy import create_engine
+
+Base = declarative_base()
```

Comment:
lets start following python style conventions for naming & other things from now on as much as possible:
https://www.python.org/dev/peps/pep-0008/#class-names

so class name below `itemProperty` should be `ItemProperty`